svgnative/src/SVGDocumentImpl.cpp

```diff
@@ -973,6 +973,128 @@ void SVGDocumentImpl::RenderElement(const Element& element, const ColorMap& colo
     SVG_ASSERT(mVisitedElements.empty());
 }
 
+Rect SVGDocumentImpl::Bounds()
+{
+    SVG_ASSERT(mGroup);
+    // TODO: Should we fire an assertion or raise exception?
```

Comment:
IIRC `SVGDocument` can only be called with an actual SVG data. So not sure if either is needed.